docs/src/getting-started/installation.md

```diff
@@ -143,6 +143,10 @@ sncast --version
 
 ## Windows
 
+> 🐧 **Info** - WSL (Windows Subsystem for Linux)
+> 
+> If you're using WSL, follow the [Linux and macOS](#linux-and-macos) instructions.
```

Comment:
Let's link this https://learn.microsoft.com/en-us/windows/wsl/install as well

And say something along the lines "you can install on windows natively but can also use [wsl]. If you are usign wsl follow this ...".